Formula/bettercap.rb

```diff
@@ -1,39 +1,31 @@
 class Bettercap < Formula
-  desc "Complete, modular, portable and easily extensible MITM framework"
+  desc "It's is the swiss army knife for network attacks and monitoring"
```

Comment:
`It's a swiss army knife for network attacks and monitoring.`

Reply:
Yes thanks